Examples/src/main/java/com/aspose/imaging/cloud/updateimages/UpdateWmfImage.java

```diff
@@ -0,0 +1,179 @@
+                FilesUploadResult result = imagingApi.uploadFile(uploadFileRequest);
+            }  finally {
+                if (localInputImageStream != null) {
+                    localInputImageStream.close();
+                }
+            }
+            String bkColor = "gray";
+            Integer pageWidth = 300;
+            Integer pageHeight = 300;
+            Integer borderX = 50;
+            Integer borderY = 50;
+            Boolean fromScratch = null;
+            // Folder with image to process. The value is null because the file is saved at the root of the storage
+            String folder = null;
+            String storage = null; // We are using default Cloud Storage
+            String exportFormat = "png";
+
+            ModifyWmfRequest modifyWmfRequest = new ModifyWmfRequest(fileName, bkColor, pageWidth, pageHeight,
+                    borderX, borderY, fromScratch, folder,
+                    storage, exportFormat);
+
+            byte[] updatedImage = imagingApi.modifyWmf(modifyWmfRequest);
+
+            // Upload updated image to cloud storage
+            String outPath = "WMFToPNG_out.png";
+            UploadFileRequest uploadFileRequest = new UploadFileRequest(outPath, updatedImage, null);
+            FilesUploadResult result = imagingApi.uploadFile(uploadFileRequest);
+        } catch (Exception e) {
+            System.out.println(e.getMessage());
+        }
+    }
+
+    /*
+     * Process existing WMF image using given parameters. Image data is passed in a request stream.
+     */
+    public void createModifiedWMFImageFromRequestBody() {
+        // Create ImagingApi instance
+        ImagingApi imagingApi = new ImagingApi(APP_KEY, APP_SID);
+
+        try {
+            File file = new File(DATA_PATH + "Sample.wmf");
+            byte[] imageData = Files.readAllBytes(file.toPath());
+
+            String bkColor = "gray";
+            Integer pageWidth = 300;
+            Integer pageHeight = 300;
+            Integer borderX = 50;
+            Integer borderY = 50;
+            Boolean fromScratch = null;
+            String outPath = null; // Path to updated file (if this is empty, response contains streamed image).
+            String storage = null; // We are using default Cloud Storage
+            String exportFormat = "png";
+
+            CreateModifiedWmfRequest modifiedWmfRequest = new CreateModifiedWmfRequest(imageData, bkColor, pageWidth,
+                                                                pageHeight, borderX, borderY, fromScratch, outPath,
+                                                                                                storage, exportFormat);
+
+            byte[] updatedImage = imagingApi.createModifiedWmf(modifiedWmfRequest);
+
+            // Save updated image to local storage
+            FileOutputStream fos = new FileOutputStream(DATA_PATH + "WMFToPNG_out.png");
```

Comment:
@MaxCrank Example code is already wrapped in try/catch block. Do you want me to add nested try/catch block around local file stream?

Reply:
@sohail-aspose sorry for the confusion caused by my typo - I've meant try/finally so that these streams are closed like at https://github.com/aspose-imaging-cloud/aspose-imaging-cloud-java/blob/SDKExamples-1/Examples/src/main/java/com/aspose/imaging/cloud/updateimages/UpdateWmfImage.java#L107

Reply:
@MaxCrank Sure.

Reply:
Done